frontend/components/GitOpsModeTooltipWrapper/_styles.scss

```
@@ -1,19 +1,23 @@
 .gitops-mode-tooltip-wrapper {
+  // Arrow styles target the nested react-tooltip via descendant selectors (mirrors the
+  // shared TooltipWrapper, which this component no longer delegates to).
+  @include tooltip5-arrow-styles;
+  display: inline-flex;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)